refinery/data_set_manager/search_indexes.py

```diff
@@ -118,46 +117,17 @@ def prepare(self, node):
                FileStoreItem.MultipleObjectsReturned) as e:
             logger.error("Couldn't properly fetch FileStoreItem: %s", e)
             file_store_item = None
-            download_url = NOT_AVAILABLE
+            download_url_or_state = constants.NOT_AVAILABLE
             data['filetype_Characteristics' + NodeIndex.GENERIC_SUFFIX] = ''
         else:
             data['filetype_Characteristics' + NodeIndex.GENERIC_SUFFIX] = \
                 file_store_item.filetype
-            download_url = file_store_item.get_datafile_url()
-            if download_url is None:
-                if not file_store_item.import_task_id:
-                    logger.debug("No import_task_id yet for FileStoreItem "
-                                 "with UUID: %s", file_store_item.uuid)
-                    download_url = PENDING
-                else:
-                    logger.debug(
-                        "FileStoreItem with UUID: %s has import_task_id: %s",
-                        file_store_item.uuid,
-                        file_store_item.import_task_id
-                    )
-                    if file_store_item.get_import_status() == SUCCESS:
-                        download_url = NOT_AVAILABLE
-                    else:
-                        # The underlying Celery code in
-                        # FileStoreItem.get_import_status() makes an assumption
-                        # that a result is "probably" PENDING even if it can't
-                        # find an associated Task. See:
-                        # https://github.com/celery/celery/blob/v3.1.20/celery/
-                        # backends/amqp.py#L192-L193 So we double check here to
-                        # make sure said assumption holds up
-                        try:
-                            TaskMeta.objects.get(
-                                task_id=file_store_item.import_task_id
-                            )
-                        except TaskMeta.DoesNotExist:
-                            logger.debug(
-                                "No file_import task for FileStoreItem with "
-                                "UUID: %s",
-                                file_store_item.uuid
-                            )
-                            download_url = NOT_AVAILABLE
-                        else:
-                            download_url = PENDING
+            download_url_or_state = file_store_item.get_datafile_url()
```

Comment:
Can we just call this `download_url` and below use `import_state`/`import_status`? It seems that `get_datafile_url()` won't return a FileStoreItem's import state, and `get_import_status()` won't return download url information.